R/select_best.R

```diff
@@ -33,6 +33,7 @@
 #' @param eval_time A numeric vector of time points where dynamic event time
 #' metrics should be chosen (e.g., the time-dependent ROC curve, etc). The
 #' values should be consistent with the values used to create `x`.
+#' @param call The call to be shown in errors and warnings.
```

Comment:
Why not use `@inheritParams rlang::args_dots_empty`?

Reply:
Because it's documented together with `select_by_*()` functions which require a parameter to be passed via the dots, see line27 and following :)

Reply:
perfect, sounds good